berik-inntektsmelding-service/src/test/kotlin/no.nav.helsearbeidsgiver.inntektsmelding.berikinntektsmeldingservice/MapInntektsmeldingKtTest.kt

```diff
@@ -50,7 +50,6 @@ class MapInntektsmeldingKtTest :
                     type shouldBe
                         Inntektsmelding.Type.Forespurt(
                             id = skjema.forespoerselId,
-                            vedtaksperiodeId = forespoersel.vedtaksperiodeId,
```

Comment:
Bør vi legge til en shouldBe-sjekk for vedtaksperiodeId på rotnivå, nå som vi fjerner denne sjekken her? Typ `vedtaksperiodeId shouldBe forespoersel.vedtaksperiodeId` på linja under der vi sjekker `mottatt` nederst i det scopet vi står i her?